CMakeLists.txt

```diff
@@ -106,6 +106,14 @@ if (NOT DEFINED COMMON_CLANG_LIBRARY_NAME)
 endif()
 set(TARGET_NAME ${COMMON_CLANG_LIBRARY_NAME}${BUILD_PLATFORM} )
 
+option(CUSTOM_PCH_EXTENSION_MAP "Set PCH extension map" OFF)
+set(PCH_EXTENSION "" CACHE STRING "")
```

Comment:
please add description message for PCH_EXTENSION

Reply:
done

Reply:
there should be a message inside set, e.g. set(PCH_EXTENSION "" CACHE STRING "Semicolon-separated list of ...")

Reply:
done